src/Symfony/Component/Validator/ConstraintViolation.php

```diff
@@ -44,12 +44,12 @@ class ConstraintViolation implements ConstraintViolationInterface
      *                                         violation
      * @param int|null        $plural          The number for determining the plural
      *                                         form when translating the message
-     * @param mixed           $code            The error code of the violation
```

Comment:
that must be an error. it cannot be mixed. \Symfony\Component\Validator\ConstraintViolationInterface::getCode says string|null. and calling code wouldn't be able to handle mixed.

Reply:
There is a test that `code` can be an int. So I couldn't add the string type. But the test is not respecting the constract as `getCode` and `setCode` both say string. But that is a topic for another time.

Reply:
I am going to prepare a PR to deprecate using anything else than a `string`.

Reply:
see #32265